src/Components/Components/src/RenderHandle.cs

```diff
@@ -21,9 +20,25 @@ internal RenderHandle(Renderer renderer, int componentId)
             _componentId = componentId;
         }
 
+        /// <summary>
+        /// Gets the <see cref="Microsoft.AspNetCore.Components.Dispatcher" /> associated with the component.
+        /// </summary>
+        public Dispatcher Dispatcher
+        {
+            get
+            {
+                if (_renderer == null)
+                {
+                    ThrowNotInitialized();
+                }
+
+                return _renderer.Dispatcher;
```

Comment:
You had a change to use throw expressions (+5 pranav points):

```C#
_renderer?.Dispatcher ?? ThrowNotInitialized();
```

Reply:
which means that method had to declare a return type.